### PR TITLE
Various: missing includes

### PR DESCRIPTION
**Pull request by petersteneteg**

### CODE REVIEW

File: modules/basegl/src/processors/multichannelraycaster.cpp [hunk 1]
```diff
@@ -32,9 +32,11 @@
 #include <modules/opengl/volume/volumegl.h>
 #include <modules/opengl/image/layergl.h>
 #include <modules/opengl/texture/textureunit.h>
+
```
Comment (martinfalk) [inline modules/basegl/src/processors/multichannelraycaster.cpp:35]: why the empty lines?

File: modules/fontrendering/src/textrenderer.cpp [hunk 2]
```diff
@@ -40,6 +40,7 @@
 #include <modules/opengl/openglutils.h>
 #include <modules/opengl/shader/shaderutils.h>
 #include <modules/opengl/texture/textureutils.h>
+#include <modules/opengl/texture/texture2d.h>                        // for Texture2D
```
Comment (martinfalk) [inline modules/fontrendering/src/textrenderer.cpp]: remove comment

**Review by martinfalk — DISMISSED** [review]

File: modules/basegl/src/processors/multichannelraycaster.cpp (hunk 1 above)
Comment (petersteneteg) [reply modules/basegl/src/processors/multichannelraycaster.cpp:35]: not sure about that

File: modules/fontrendering/src/textrenderer.cpp (hunk 2 above)
Comment (petersteneteg) [reply modules/fontrendering/src/textrenderer.cpp]: We might add these comment for every include in the future, or not, we should discuss that.

**Review by ResearchDaniel — DISMISSED** [review]

**Review by ResearchDaniel — APPROVED** [review]

**Review by martinfalk — APPROVED** [review]

**Pull request merged**